recipes/libgit2/pstdint.h

```diff
@@ -0,0 +1,912 @@
+/*  A portable stdint.h
```

Comment:
Sorry that cost you a lot of time. Hopefully this will save you time in the future.

Probably, but not sure off the top of my head. Though this seems like good material for the [docs]( https://github.com/conda-forge/conda-forge.github.io/tree/master/docs ). Please feel free to add stuff there I'm sure you have a lot of lessons learned that may be very helpful for people at this point.